src/app/components/common/nav/nav.component.html

```diff
@@ -1,3 +1,32 @@
-<div >
-
-</div>
\ No newline at end of file
+<nav class="navbar navbar-expand-lg navbar-light">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">Dev<span class="text-muted">Hitch</span></a>
+    <button
```

Comment:
Use inline code for the whole button! i know it is based on dev preference, but when the component grows it becomes troublesome as the lines increases

Reply:
@banujan6

Reply:
@sajeetharan but it comes with code formatter. I think it's easier to read rather than horizontal scrolling